README.md

```diff
@@ -26,19 +26,19 @@
 
 <!-- PROJECT DESCRIPTION -->
 
```

Comment:
- [ ] Kindly implement the action dispatch for displaying "Under construction" on the Category page. When I click the button I should see the text.